src/appmixer/activecampaign/bundle.json

```diff
@@ -1,7 +1,15 @@
 {
     "name": "appmixer.activecampaign",
-    "version": "1.0.1",
-    "changelog": [
-        "Initial version"
-    ]
+    "version": "1.0.2",
+    "changelog": {
+        "1.0.0": [
+            "Initial version"
+        ],
+        "1.0.2": [
+            "CreateDeal: Deal amount and Currency are now required.",
```

Comment:
making fields required is a breaking change (it should be v2.0.0)

Reply:
I understand but what if the component didnt work without it before? I mean you couldnt create a deal without Deal amount and Currency